unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/WearableItem.cs

```diff
@@ -182,6 +182,16 @@ public static HashSet<string> CompoundHidesList(string bodyShapeId, List<Wearabl
 
         return result;
     }
+
+    //Workaround to know the net of a wearable. 
+    //Once wearables are allowed to be moved from Ethereum to Polygon this method wont be reliable anymore
+    //To retrieve this properly first we need the catalyst to send the net of each wearable, not just the ID
+    public bool IsInL2()
+    {
+        if (id.Contains("matic") || id.Contains("mumbai"))
```

Comment:
Is there any more specific way to check this?
I'm worry that if we already have a "Matic" themed wearable on the Ethereum chain with the name matic on it id I would report a false positive
Maybe something like StartWith("urn:blablabla")? I don't know, I'm just guessing :P

Reply:
I could use `urn:decentraland:{protocol}` where protocol is `matic` or `mumbai`. Does that feel better?

Reply:
That would be great. It should solve the possible situation that I was worry about 👍

Reply:
Done!